pages/releases/4.3.html

```diff
@@ -0,0 +1,2109 @@
+				<div id="gdscript-binary-tokenization-on-export" class="release-card inverted media-top span-2">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#gdscript-binary-tokenization-on-export">Binary tokenization on
+										export</a>
+								</h4>
+								<div class="c-blockquote">
+									<span class="highlight">Lean</span> and mean
+								</div>
+								<div class="c-content">
+									<p>
+										With this new export option, you can reduce the size of the exported
+										GDScript code. But it also offers a way to obfuscate your source code.
+									</p>
+									<p>
+										While it doesn't protect your work from being decompiled, it at least hides
+										your source code from being available publicly.
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="vnen"
+									data-read-more="https://github.com/godotengine/godot/pull/87634">
+								</div>
+							</div>
+						</div>
+						<div class="release-card-media">
+							<img class="release-card-image" src="/storage/releases/4.3/images/export-binary-tokens.webp"
+								srcset="
+									/storage/releases/4.3/images/export-binary-tokens-2x.webp 2x
+								" alt="Image of the export GDScript binary tokenization settings." draggable="false" />
+						</div>
+					</div>
+				</div> <!-- #gdscript-binary-tokenization-on-export -->
+
+				<div id="gdscript-is-not-operator" class="release-card">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#gdscript-is-not-operator">
+										<code>
+											<span class="keyword">is not</span>
+										</code>
+										operator
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										<code class="highlight">
+											<span class="controlflow">if</span>
+											<span class="keyword">not</span>
+											my_node
+											<span class="keyword">is</span>
+											<span class="basetype">Node3D</span><span class="symbol">:</span>
+										</code>
+									</p>
+									<p>
+										It isn't much readable, huh? This line tests if
+										<code class="highlight">my_node</code>
+										is <em>not</em> of type
+										<code class="highlight">
+											<span class="basetype">Node3D</span>
+										</code>.
+									</p>
+									<p>
+										You can now type this instead, which is really closer syntax wise from what
+										we actually say and mean:<br>
+										<code class="highlight">
+											<span class="controlflow">if</span>
+											my_node
+											<span class="keyword">is not</span>
+											<span class="basetype">Node3D</span><span class="symbol">:</span>
+										</code>
+									</p>
+								</div>
```

Comment:
```suggestion
									<a href="#gdscript-is-not-operator">
										<code>
											<span class="keyword">is not</span>
										</code>
										operator
									</a>
								</h4>
								<div class="c-content">
									<p>
									        Instead of writing <br>
										<code class="highlight">
											<span class="controlflow">if</span>
											<span class="keyword">not</span>
											my_node
											<span class="keyword">is</span>
											<span class="basetype">Node3D</span><span class="symbol">:</span>
										</code>
									</p>
									<p>
									        You can now simply write <br>
										<code class="highlight">
											<span class="controlflow">if</span>
											my_node
											<span class="keyword">is not</span>
											<span class="basetype">Node3D</span><span class="symbol">:</span>
										</code>
									</p>
									<p>
									This rather small change should help keep your code readable, by being closer to the natural language equivalent of what you are attempting to do: the test checks whether my_node <code>is not</code> of the type Node3D.
									</p>
								</div>
```